python_modules/dagster/dagster/core/definitions/pipeline.py

```diff
@@ -37,6 +37,20 @@ class PipelineDefinition(object):
         Solids within a pipeline are arranged as a DAG (directed, acyclic graph). Dependencies
         determine how the values produced by solids flow through the DAG.
 
+    Args:
+        solids (List[SolidDefinition]):
```

Comment:
So the only place where SolidInstances are used is in dependencies. That is not particuarily clear from this

Reply:
I think this makes things more clear https://github.com/dagster-io/dagster/pull/1241